Modules/Filtering/ImageIntensity/test/itkVectorExpandImageFilterTest.cxx

```diff
@@ -272,57 +272,25 @@ itkVectorExpandImageFilterTest(int, char *[])
     ++streamIter;
   }
 
-  if (!testPassed)
-  {
-    std::cout << "Test failed." << std::endl;
-    return EXIT_FAILURE;
-  }
-
   // Test error handling
+  std::cout << "Setting Input to nullptr" << std::endl;
 
-  try
-  {
-    testPassed = false;
-    std::cout << "Setting Input to nullptr" << std::endl;
-    expander->SetInput(nullptr);
-    expander->Update();
-  }
-  catch (const itk::ExceptionObject & err)
-  {
-    std::cout << err << std::endl;
-    expander->ResetPipeline();
-    expander->SetInput(input);
-    testPassed = true;
-  }
+  expander->SetInput(nullptr);
+  ITK_TRY_EXPECT_EXCEPTION(expander->Update());
 
-  if (!testPassed)
-  {
-    std::cout << "Test failed." << std::endl;
-    return EXIT_FAILURE;
-  }
 
+  expander->ResetPipeline();
+  expander->SetInput(input);
 
-  try
-  {
-    testPassed = false;
-    std::cout << "Setting Interpolator to nullptr" << std::endl;
-    expander->SetInterpolator(nullptr);
-    expander->Update();
-  }
-  catch (const itk::ExceptionObject & err)
-  {
-    std::cout << err << std::endl;
-    expander->ResetPipeline();
-    expander->SetInterpolator(interpolator);
-    testPassed = true;
-  }
+  std::cout << "Setting Interpolator to nullptr" << std::endl;
+  expander->SetInterpolator(nullptr);
+  ITK_TRY_EXPECT_EXCEPTION(expander->Update());
+
+
+  expander->ResetPipeline();
+  expander->SetInterpolator(interpolator);
```

Comment:
It is weird that I had to add these two lines after the call to update the filter. The previous commit
https://github.com/InsightSoftwareConsortium/ITK/commit/10b5e6a506d3291438552c966833871530c6dc9a

produced a test failure:
https://open.cdash.org/viewTest.php?onlyfailed&buildid=8953052

It is unclear to me the reason of such failure, despite the test completing its instructions. Even more so this being a seemingly harmless style change. Any thoughts?